Dart/src/com/jetbrains/lang/dart/ide/actions/DartEditorNotificationsProvider.java

```diff
@@ -105,11 +105,14 @@ public EditorNotificationPanel createNotificationPanel(@NotNull final VirtualFil
   private static class PubActionsPanel extends EditorNotificationPanel {
     private PubActionsPanel() {
       super(EditorColors.GUTTER_BACKGROUND);
-      myLinksPanel.add(new JLabel("Pub actions:"));
+
+      icon(DartIcons.Dart_16);
+      text("Pub actions");
+
```

Comment:
I used `myLinksPanel` instead of `text()` to address layout on wide monitors. And used no icon, because it looks strange when not aligned to the left. Which one do you like more:
![image](https://cloud.githubusercontent.com/assets/4499789/20886504/547d12f8-bb07-11e6-8c7f-7840912a1901.png)
![image](https://cloud.githubusercontent.com/assets/4499789/20886416/fd027022-bb06-11e6-8cbf-f828292702e8.png)